rmgpy/rmg/model.py

```diff
@@ -1072,15 +1073,18 @@ def thermoFilterDown(self,maximumEdgeSpecies,minSpeciesExistIterationsForPrune=0
 
             ind = 0
             removeInds = []
-                
+            
+            
             while ind < len(inds) and numToRemove > 0: #find the species we can remove and collect indices for removal     
                 i = inds[ind]
                 spc = spcs[i]
-                if iteration - spc.creationIteration > minSpeciesExistIterationsForPrune:
+                if iteration - spc.creationIteration >= minSpeciesExistIterationsForPrune:
                     removeInds.append(i)
                     numToRemove -= 1
                 ind += 1
-                    
+            
+            logging.debug('found {0} eligible species for filtering'.format(len(removeInds)))
```

Comment:
maybe the commit with these debug statements could be squashed with the thermoFilterDown method itself?